pkg/envtest/webhook.go

```diff
@@ -377,7 +377,7 @@ func readWebhooks(path string) ([]runtime.Object, []runtime.Object, error) {
 
 			switch {
 			case generic.Kind == "MutatingWebhookConfiguration":
-				if generic.APIVersion != "v1beta1" && generic.APIVersion != "v1" {
+				if generic.APIVersion != "admissionregistration.k8s.io/v1beta1" && generic.APIVersion != "admissionregistration.k8s.io/v1" {
```

Comment:
could you use the comparison @DirectXMan12 suggested here? https://github.com/kubernetes-sigs/controller-runtime/issues/867 instead of string